processor/runtime/src/main/java/com/redhat/service/smartevents/processor/errorhandler/ErrorPublisher.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.service.smartevents.processor.errorhandler;
+
+public interface ErrorPublisher {
```

Comment:
Yeah, yeah, the name of this and related classes isn't great.

The intent is however there; to have a POJO containing back-channel meta-messages.

These can be errors or informational (or warnings, if that's even possible).

Reply:
Deleted with the commit to use Kafka's DLQ support.